src/resolve/resolved-dns-server.c

```diff
@@ -959,14 +985,16 @@ void dns_server_dump(DnsServer *s, FILE *f) {
                 "\tSeen truncated packet: %s\n"
                 "\tSeen OPT RR getting lost: %s\n"
                 "\tSeen RRSIG RR missing: %s\n"
-                "\tSeen invalid packet: %s\n",
+                "\tSeen invalid packet: %s\n"
+                "\tSeen DO flag not copied: %s\n",
```

Comment:
This message doesn't read very well to me - what about: "DO flag received: yes/no"

Reply:
"DO flag received" is not clear enough: we don't know *why* it wasn't copied. Maybe "Server dropped DO flag"?